tests/simulate/test_simulate.py

```diff
@@ -1,13 +1,83 @@
 """Unit tests for simulate module."""
+import tempfile
 import unittest
 
 import numpy as np
 
 import relentless
 
+class test_Simulation(unittest.TestCase):
+    """Unit tests for relentless.Simulation"""
+
+    #mock function for use as operation
+    class OperationA(relentless.simulate.SimulationOperation):
```

Comment:
I would replace these with two ops, like you have in `test_dilute.py`, that use the simple attribute setting I suggested there.